Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Umbraco-Interface/index.md

```diff
@@ -1,28 +1,58 @@
+The last button has three dots, this gives you the following options: 
+- Getting Started
+- Set up a Project
+- Deployment
+- Troubleshooting
 
-By default, on a desktop with a large screen size, the tree view is always visible giving you quick access
-to select and navigate website content. On smaller devices such as laptops and tablets, the tree view will be closed and only opens after you click on one of the navigation icons in the section bar on the left of the screen.
+![Environment.png](images/Search-and-help.png)
 
-After selecting an item within the tree view on a smaller device, it will automatically close to save screen space. If you wish to close the tree view without selecting a page, you can move your mouse out of the tree view, or alternatively on a touch device you can swipe left over the tree view. Below are examples of how the interface will look on a tablet.
 
-![smallScreenDashboard.jpg](images/smallScreenDashboard.jpg)
+### 3. The three buttons in the top right
+ The first button is a search field here you can search for the content that you are looking for.
+The second button is the help button if you click this you will get options for **help**.
+The third button is your **profile**, this will give you the option to edit your profile and password.
 
-![dashboardSlideOut.jpg](images/dashboardSlideout.jpg)
+![Environment.png](images/Two-tabs-left.png)
+
+### 4. The two buttons slightly towards top left
+The first button is **Your Workspace** that takes you to the tab with the circles on it.
+The second button is **Redirect URL Management** this is for redirects on you project.
+
+![Environment.png](images/The-top-bar-right.png)
+
+### 5. The eight buttons in the far top left
+These buttons are the tabs that you will be working under during you work with you project. 
+The first button is **Content** this is where you manage your content.
+The next button is **Media**, under this tab you can manage images and other media files.
+The third button is **Settings**, in this tab you handle your meta data such as document types.
+Button number four is **Packages**, this tab lets you manage and install packages.
+The fifth button is **Users**, this tab lets you manage the users on the project.
+Button six is the **Members** tab, and here you can handle the members of the project.
+The seventh button in the row is **Forms** tab, here you can create and manage your forms.
+The final and eight button is **Translation**, this tab is for managing languages.  
+
+![Environment.png](images/Colone-left.png)
+
+### 6. The two buttons on the far left
+The first one is **Content** this allows you to create content nodes, and manage your content tree, if you hover over it then it will highlight with a darker color indicating that you are hovering over it, and a button with three dots will show up, clicking this will present you with additional options, there will be other option if you right click the field.
```

Comment:
It would be great with some general details about trees here! 😁 

Also, it's great explaining the options in the tree.. But as far as I can see, these options are all also covered in the section on creating content: https://our.umbraco.com/documentation/Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/ 🤔 
Finally: The sentence is more than 40 words 🙈 Any chance you could split it up or make it shorter? (https://our.umbraco.com/documentation/Contribute/Style-Guide/#long-sentences)